test/test_rigidbody.cpp

```diff
@@ -330,7 +330,7 @@ TEST(Joints, unitTest)
         biorbd::rigidbody::GeneralizedVelocity Qdot(model);
 
         biorbd::utils::Vector3d angularMomentum(joints.angularMomentum(Q, Qdot));
```

Comment:
Could you put Q and Qdot not equal to zero, it is hard to determine if angleMomentum == 0 is 0 because it is the right answer, VS it does nothing!

Reply:
Done :)

Reply:
Wow, I almost let it passed, but decided to do it myself before Release a new version of biorbd... Turns out, there was a bug for this function! It wasn't taking into account the Qdot...
Thanks for helping spotting that!

Reply:
Do you want me to change anything here?

Reply:
Happy you were able to spot it! I really did not know.

Reply:
I changed it so nothing to do.
Me too! This is a very important computation for my project haha!